### DISCUSSION TITLE
Separate Photo and Video from LOGB

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#498] [Feature Requests]
Hi,

I set LOGB=400 in my GoPro 11 and I notice that it spills over to the Photo as well. Can we restrict that LOGB only works for video NOT on the photo or time-lapse?

**Comment by dnewman-gpsw** [comment]
Unfortunately LOGB be is such a deep hack, it is not aware of the current camera mode.  This is doable, but not straight forward.  Added to the wish list.

**Reply by tetrahydra** [reply]
Thanks. It would be really great! Editing a LOGB JPEG is not fun. Similar to a time-lapse video.
